src/lib/core/backtrace.c

```diff
@@ -57,29 +57,34 @@ collect_current_stack(struct backtrace *bt, void *stack)
 	unw_context_t unw_ctx;
 	int rc = unw_getcontext(&unw_ctx);
 	if (rc != 0) {
-		say_error("unwinding error: unw_getcontext failed");
+		/*
+		 * XXX: turn this and other say_debug occurrences back to
+		 * say_error once the error stops being spammed on mac.
+		 * (https://github.com/tarantool/tarantool/issues/7980).
+		 */
```

Comment:
I don't think we should ever revert it, because there may be other errors resulting in libunwind failures. I think that switching permanently to `say_debug` is fine, because if there's an error in libunwind, the user will see it, because backtraces will be missing. In this case, they may raise the logging level to 'debug' to find out the error reason.

Reply:
Ok, I don't mind.